README.md

```diff
@@ -116,14 +137,27 @@ Run it!
 
 docker run -it --network host rmf:latest bash -c "export ROS_DOMAIN_ID=9; ros2 launch rmf_demos_gz office.launch.xml headless:=1"
 ```
+
 This will run `rmf_demos` in headless mode. Open [this link](https://open-rmf.github.io/rmf-panel-js/) with a browser to start a task.
 
 (Experimental) User can also run `rmf_demos` in “non-headless” graphical form, via [rocker](https://github.com/osrf/rocker).
 
-## Roadmap
-
-A near-term roadmap of the entire OpenRMF project (including and beyond `rmf_traffic`) can be found in the user manual [here](https://osrf.github.io/ros2multirobotbook/roadmap.html).
-
-## Integrating with RMF
-
-Instructions on how to integrate your system with OpenRMF can be found [here](https://osrf.github.io/ros2multirobotbook/integration.html).
+## Package Information
```

Comment:
Can we add link to the specific version tag instead of the general repo? 

It would also be good if we can add what distros (ubuntu 20.04 or whaterver) these packages can be installed in and maybe their package name in that distro.

Reply:
Not all linux distros have the same packages of a certain RMF distro, ie: some packages are available on ubuntu and not in rehat

Reply:
Okay got it. I'll make the necessary changes.